Library/ViewModels/DiscoveryFiltersViewModelTests.swift

```diff
@@ -433,4 +466,44 @@ internal final class DiscoveryFiltersViewModelTests: TestCase {
       "Server did not advance, categories loaded from cache."
     )
   }
+
+  /*
+   Helper for testing discoverProperties All Project Filter
+   */
+
+  private func assertDiscoveryPropertiesForAllProject(_ props: [String: Any]?) {
```

Comment:
I think you have the right idea here but this is very specific to one case and for that reason I don't think it should be a specifically defined method.